pages/editor.vue

```diff
@@ -54,6 +55,18 @@
         addElementToActivePanel(tempEl);
     }
 
+    function upZIndex(eId) {
+        comic.getPage(0).getStrip(0).getPanel(activePanelIndex.value).moveZIndexUp(eId);
+    }
+
+    function downZIndex(eId) {
+        comic.getPage(0).getStrip(0).getPanel(activePanelIndex.value).moveZIndexDown(eId);
+    }
+
+    function switchZIndex(obj) {
+        comic.getPage(0).getStrip(0).getPanel(activePanelIndex.value).switchZIndexBetweenTwoElement(obj.eId1, obj.eId2);
+    }
+
```

Comment:
Generally I think, that if we need to handle move up and down logic here already also the switching out of `WrapperCanvas` to reduces scattering the same logic across multiple places.

Reply:
Problem is, in my opinion: The new layout system is composed of a lot of element that are coming from different places.  I am not a huge fan of emitting / chain-passing over 5+ components.
But I dont understand fully your comment.

Reply:
After thinking I agree.